com.github.PintaProject.Pinta.yaml

```diff
@@ -27,6 +27,7 @@ finish-args:
   - --filesystem=/run/media
   # GVfs access
   - --filesystem=xdg-run/gvfs:ro
+  - --filesystem=xdg-run/gvfsd:ro
```

Comment:
@hfiguiere @barthalion I got warnings that I should add this line. Is this the right way to handle gvfs?

Reply:
the details are there https://github.com/flathub/flathub/issues/2180

Reply:
and Pinta is on the list

Reply:
@hfiguiere Thanks for the link! So I am doing it correctly now and it can be marked as fixed once Pinta 1.8 got released?